### ISSUE TITLE
Connection terminated (4014, 'Disallowed intent(s).')

### ISSUE THREAD

**Issue by Vitaphobiac** [herrozerro/Cypher-Bot#9]
After adding my bot, updating appsettings.json with my token.
Executed Cypherbot.sln and ran in this issue: "Connection terminated (4014, 'Disallowed intent(s).')"
Have also tried setting "appInitialize" to false, with no luck

I am incredibly inexperienced, but I'd really appreciate if you can help me somehow.

**Comment by herrozerro** [comment]
What permissions did you give your bot?

**Comment by herrozerro** [comment]
What permissions did you give your bot?  

You may need message content intent

**Comment by Vitaphobiac** [comment]
Thanks for replying so quickly!
First I've added the bot on Discord with only the scope 'bot', since that is mentioned in the README.
Tried to run the code, got the error message.
Then I've added the bot under the scope 'bot' as before, and with permissions 'administrator'.

Because of your message I looked around a bit more, and yes, it was indeed message content intent.

Thank you again for giving me the motivation to look around more!
